README.md

```diff
@@ -1,3 +1,11 @@
 cppcheclipse is an Eclipse plugin which integrates [cppcheck](http://sourceforge.net/projects/cppcheck/) with the [CDT project](https://eclipse.org/cdt/). You can run/configure cppcheck from the Eclipse UI.
 
+To build the project by hand it requires maven and Linux platform (maven under Windows will not compile the project properly). To compile run following commands:
+```bash
+cd com.googlecode.cppcheclipse.parent
+mvn clean verify
+ls ../com.googlecode.cppcheclipse.repository/target/*.zip -l
```

Comment:
why is `ls` necessary on the zip file?

Reply:
It's not necessary, but for somebody who maybe doesn't know where the artifact is stored this will show the location, date and size which sometimes is handy even when knowing where the result will be.

Reply:
Ok, since this is not building I would rather add a sentence below like
`You can find the built p2 repository now in ../com.googlecode.cppcheclipse.repository/target in zip format`

Reply:
Sure